packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/ecs/run-task.ts

```diff
@@ -436,26 +450,33 @@ export class EcsRunTask extends sfn.TaskStateBase implements ec2.IConnectable {
   }
 }
 
-function renderOverrides(containerOverrides?: ContainerOverride[]) {
-  if (!containerOverrides || containerOverrides.length === 0) {
+function renderOverrides(cpu?: string, memoryMiB?: string, containerOverrides?: ContainerOverride[]) {
```

Comment:
Given that there are [many more props](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_TaskOverride.html) to be overriden, shouldn't we use an interface to pass these arguments instead of positional arguments?

e.g. 
```ts
function renderOverrides(props: OverrideProps) 

interface OverrideProps {
 cpu?: number;
 memoriMib?: number;
 containerOverrides?: ContainerOverride[];
}
```

Reply:
Thank you. I'll fix it.

Reply:
done 8bb2d45